packages/nodes-base/nodes/Mattermost/v1/transport/index.ts

```diff
@@ -34,7 +34,7 @@ export async function apiRequest(this: IHookFunctions | IExecuteFunctions | ILoa
 	const credentials = this.getCredentials('mattermostApi');
 
 	if (credentials === undefined) {
-		throw new NodeOperationError(this.getNode(), 'No credentials got returned!');
+		throw new NodeOperationError(this.getNode(), 'No credentials returned!');
```

Comment:
We could refactor `this.getCredentials` to throw this error inside, so that we do not have to repeat this check in each and every node that has credentials.